cmd/restic/cmd_restore.go

```diff
@@ -199,6 +232,31 @@ func runRestore(ctx context.Context, opts RestoreOptions, gopts GlobalOptions,
 
 	includePatterns := filter.ParsePatterns(opts.Include)
 	insensitiveIncludePatterns := filter.ParsePatterns(opts.InsensitiveInclude)
+
+	if len(opts.IncludeFiles) > 0 {
```

Comment:
Please create a `includePatternOptions` struct similar to `excludePatternOptions`. The `includeByNameFunc`s must be somewhat different in that it has to use `filter.ListWithChild(includePatterns, item)` and return `matched, childMayMatch, err`.

Reply:
This is done. I added a file called `include.go`